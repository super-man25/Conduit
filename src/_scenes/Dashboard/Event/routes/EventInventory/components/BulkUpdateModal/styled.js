import styled, { keyframes } from 'styled-components';
import { cssConstants } from '_constants';
import { Box, Text, H4 } from '_components';

export const slideUpAnimation = keyframes`
  0% {
    opacity: 0;
    transform: translate(-50%, -20%);
  }
  100% {
    opacity: 1;
  }
`;

export const ModalOverlay = styled.div`
  position: fixed;
  top: 0;
  bottom: 0;
  left: 0;
  right: 0;
  opacity: 0.3;
  background-color: ${cssConstants.PRIMARY_LIGHT_BLACK};
`;

export const ModalContent = styled.div`
  position: fixed;
  top: 50%;
  left: 50%;
  transform: translate(-50%, -50%);
  width: 350px;
  z-index: 9;
  box-shadow: 0 1px 10px rgba(0, 0, 0, 0.27);
  animation: ${slideUpAnimation} 200ms ease-out;
  animation-fill-mode: forwards;
`;

export const ModalHeader = styled.div`
  padding: 1.5rem;
  background-color: ${cssConstants.PRIMARY_LIGHTEST_GRAY};
  border-bottom: 1px solid ${cssConstants.PRIMARY_BLUE};
  border-top-left-radius: 3px;
  border-top-right-radius: 3px;
`;

export const ModalBody = styled.div`
  padding: 1.5rem;
  background-color: ${cssConstants.PRIMARY_WHITE};
  border-bottom-left-radius: 3px;
  border-bottom-right-radius: 3px;
`;

export const Title = styled(H4)`
  margin: 0;
  margin-bottom: 0.25rem;
  color: ${cssConstants.PRIMARY_LIGHT_BLACK};
`;

export const NumberInput = styled.input`
  background-color: ${cssConstants.PRIMARY_WHITE};
  border: 1px solid ${cssConstants.PRIMARY_GRAY};
  border-radius: 3px;
  box-sizing: border-box;
  height: 40px;
  margin: 0;
  padding: 0 0.5rem;
  width: 100%;
`;

export const Label = styled.label`
  display: block;
  margin-bottom: 0.5rem;
  color: ${cssConstants.PRIMARY_LIGHT_BLACK};
`;

export const Field = styled(Box)`
  position: relative;
`;

export const FieldErrorText = styled(Text)`
  position: absolute;
  top: 100%;
`;
