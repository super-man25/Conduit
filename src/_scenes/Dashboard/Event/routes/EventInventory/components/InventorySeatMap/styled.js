import React from 'react';
import { Box, CenteredLoader } from '_components';
import styled from 'styled-components';

export const EventInventoryVerticalRule = styled(Box)`
  background-color: #979797;
  cursor: pointer;
`;

export const SeatIcon = styled.img`
  cursor: pointer;
`;

export const SeatMapLoader = () => (
  <div style={{ position: 'relative', height: '100%', minHeight: 400 }}>
    <CenteredLoader />
  </div>
);
