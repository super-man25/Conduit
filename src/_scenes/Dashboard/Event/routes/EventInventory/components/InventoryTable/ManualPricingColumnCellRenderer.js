// @flow
import * as React from 'react';
import { Text, Flex, ClearIcon } from '_components';
import { cssConstants } from '_constants';
import { ManualPricingButton, ManualPricingInput } from './styled';
import { connect } from 'react-redux';
import { actions } from '_state/eventInventory';
import { formatUSD } from '_helpers/string-utils';

type State = {
  overridePrice: number | string
};

type Props = {
  isEditing: boolean,
  setManualPrice: (value: number | string) => void,
  manualPriceLocked: boolean,
  setEditingManualPrice: () => void,
  cancelEditingManualPrice: () => void,
  rowData: { [string]: any },
  cellData: any
};

export class ManualPricingCellPresenter extends React.Component<Props, State> {
  state = { overridePrice: '' };

  startEditingPrice = () => {
    const { setEditingManualPrice, manualPriceLocked } = this.props;

    if (!manualPriceLocked) {
      this.setState(
        { overridePrice: this.props.rowData.overridePrice || '' },
        setEditingManualPrice
      );
    }
  };

  cancelEditingPrice = () => {
    const { cancelEditingManualPrice } = this.props;
    cancelEditingManualPrice();
  };

  setPrice = () => {
    const { overridePrice } = this.state;
    this.props.setManualPrice(overridePrice);
  };

  onOverridePriceChanged = (e: SyntheticEvent<HTMLInputElement>) => {
    const { value } = (e.currentTarget: HTMLInputElement);
    this.setState({ overridePrice: value });
  };

  render() {
    const { overridePrice } = this.state;
    const {
      rowData: { listedPrice },
      cellData,
      isEditing,
      manualPriceLocked
    } = this.props;

    if (!isEditing) {
      const textOpacity = manualPriceLocked ? 0.5 : 1;
      const textToDisplay = cellData !== null ? 'EDIT PRICE' : 'SET PRICE';

      return (
        <Flex align="center">
          <Text
            size={14}
            color={cssConstants.PRIMARY_LIGHT_BLUE}
            onClick={this.startEditingPrice}
            opacity={textOpacity}
          >
            {textToDisplay}
          </Text>
          {cellData && (
            <Text
              size={14}
              color={cssConstants.PRIMARY_LIGHT_BLACK}
              marginLeft="1rem"
            >
              {formatUSD(cellData)}
            </Text>
          )}
        </Flex>
      );
    }

    const inputPlaceholder =
      cellData !== null ? formatUSD(cellData) : formatUSD(listedPrice);

    return (
      <Flex align="center">
        <ManualPricingInput
          type="text"
          pattern="[0-9]"
          placeholder={inputPlaceholder}
          value={overridePrice}
          onChange={this.onOverridePriceChanged}
        />
        <ManualPricingButton onClick={this.setPrice}>
          Set Price
        </ManualPricingButton>
        <ClearIcon
          width={16}
          height={16}
          styles={{ marginLeft: 7, cursor: 'pointer' }}
          onClick={this.cancelEditingPrice}
        />
      </Flex>
    );
  }
}

const mapStateToProps = (
  { eventInventory: { manualPriceEditId } },
  ownProps
) => {
  return {
    manualPriceLocked: manualPriceEditId !== null,
    isEditing: manualPriceEditId === ownProps.rowData.id
  };
};

const mapDispatchToProps = (dispatch, ownProps) => ({
  setEditingManualPrice: () =>
    dispatch(actions.setEditingManualPrice(ownProps.rowData.id)),
  setManualPrice: (value) =>
    dispatch(actions.setEventRowManualPrice(ownProps.rowData.id, value)),
  cancelEditingManualPrice: () => dispatch(actions.cancelEditingManualPrice())
});

const ManualPricingCell = connect(mapStateToProps, mapDispatchToProps)(
  ManualPricingCellPresenter
);

export const manualPricingColumnCellRenderer = (props: any) => (
  <ManualPricingCell {...props} />
);
