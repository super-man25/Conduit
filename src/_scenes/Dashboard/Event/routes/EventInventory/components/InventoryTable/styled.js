import styled from 'styled-components';
import { cssConstants } from '_constants';
import { Button, Input, Flex, Box } from '_components';

export const ManualPricingInput = Input.extend`
  box-sizing: border-box;
  margin: 0 0.5rem 0 0;
  padding: 10px 16px;
  max-height: 35px;
  max-width: 80px;
`;

export const ManualPricingButton = Button.extend`
  box-sizing: border-box;
  margin: 0;
  max-width: 80px;
  max-height: 35px;
`;

export const TableHeaderCell = styled.div`
  text-align: ${(props) => props.align || 'left'};
  color: ${cssConstants.PRIMARY_DARKEST_GRAY};
  font-weight: 500;
`;

export const IconContainer = Flex.extend`
  display: inline-flex;
  cursor: pointer;
  position: absolute;
  left: 0;
  top: 50%;
  transform: translateY(-50%);
  opacity: 0.667;
`;

export const PositionedBox = Box.extend`
  position: ${(props) => props.position};
  top: ${(props) => props.top};
  bottom: ${(props) => props.bottom};
  left: ${(props) => props.left};
  right: ${(props) => props.right};
  z-index: ${(props) => props.zIndex};
`;

export const ScaleFilterContainer = Box.extend`
  width: 280px;
  border: 1px solid ${cssConstants.SECONDARY_BLUE};
  background-color: ${cssConstants.PRIMARY_WHITE};
  position: relative;
  box-shadow: 0 2px 5px 0px rgba(0, 0, 0, 0.43);

  &::before {
    content: '';
    width: 8px;
    height: 8px;
    position: absolute;
    top: -5px;
    left: 43px;
    transform: rotate(45deg);
    z-index: 1;
    background-color: ${cssConstants.PRIMARY_WHITE};
    border-top: 1px solid ${cssConstants.SECONDARY_BLUE};
    border-left: 1px solid ${cssConstants.SECONDARY_BLUE};
  }
`;

export const ScaleFilterList = styled.ul`
  margin: 0;
  height: 100%;
  list-style: none;
  padding: 0;
  overflow: auto;
`;

export const ScaleFilterListItem = styled.li`
  height: 35px;
  display: flex;
  padding: 0 1rem;
  align-items: center;
`;

export const Checkbox = styled.input.attrs({ type: 'checkbox' })`
  margin-right: 0.5rem;
`;

export const ScaleFilterButton = styled.button`
  background-color: ${cssConstants.PRIMARY_LIGHT_BLUE};
  border: none;
  border-radius: 2px;
  color: ${cssConstants.PRIMARY_WHITE}
  font-weight: 300;
  height: 25px;
  min-width: 60px;
  padding: 0 1rem;
`;
