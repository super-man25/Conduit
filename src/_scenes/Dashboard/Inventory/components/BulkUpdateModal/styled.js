import styled from 'styled-components';
import { cssConstants } from '_constants';
import { Box, Text, H4, Checkbox } from '_components';

export const ModalHeader = styled.div`
  padding: 1.5rem;
  background-color: ${cssConstants.PRIMARY_LIGHTEST_GRAY};
  border-bottom: 1px solid ${cssConstants.PRIMARY_BLUE};
  border-top-left-radius: 3px;
  border-top-right-radius: 3px;
`;

export const ModalBody = styled.div`
  padding: 1.5rem;
  background-color: ${cssConstants.PRIMARY_WHITE};
  border-bottom-left-radius: 3px;
  border-bottom-right-radius: 3px;
`;

export const Title = styled(H4)`
  margin: 0;
  margin-bottom: 0.25rem;
  color: ${cssConstants.PRIMARY_LIGHT_BLACK};
`;

export const NumberInput = styled.input`
  background-color: ${cssConstants.PRIMARY_WHITE};
  border: 1px solid ${cssConstants.PRIMARY_GRAY};
  border-radius: 3px;
  height: 40px;
  margin: 0;
  padding: 0 0.5rem;
  width: 100%;
`;

export const Label = styled.label`
  display: block;
  margin-bottom: 0.5rem;
  color: ${cssConstants.PRIMARY_LIGHT_BLACK};
`;

export const Field = styled(Box)`
  position: relative;
`;

export const FieldErrorText = styled(Text)`
  position: absolute;
  top: 100%;
`;

export const StyledCheckbox = styled(Checkbox)`
  margin-top: 15px;
`;
