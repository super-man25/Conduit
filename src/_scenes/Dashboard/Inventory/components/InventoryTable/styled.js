import styled from 'styled-components';
import { colors } from '_constants';
import { PrimaryButton, Input, Flex, Box } from '_components';

export const ManualPricingInput = styled(Input)`
  margin: 0 0.5rem 0 0;
  padding: 10px 8px;
  max-height: 35px;
  max-width: 80px;
`;

export const ManualPricingButton = styled(PrimaryButton)`
  margin: 0;
  min-width: 50px;
  width: 80px;
  max-height: 35px;
  padding: 0.5rem;
`;

export const TableHeaderCell = styled.div`
  text-align: ${(props) => props.align || 'left'};
  color: ${colors.gray};
  font-weight: 500;
`;

export const IconContainer = styled(Flex)`
  display: inline-flex;
  cursor: pointer;
  position: absolute;
  left: 0;
  top: 50%;
  transform: translateY(-50%);
  opacity: 0.667;
`;

export const PositionedBox = styled(Box)`
  position: ${(props) => props.position};
  top: ${(props) => props.top};
  bottom: ${(props) => props.bottom};
  left: ${(props) => props.left};
  right: ${(props) => props.right};
  z-index: ${(props) => props.zIndex};
`;

export const FilterContainer = styled(Box)`
  width: 280px;
  border: 1px solid ${colors.blue};
  background-color: ${colors.white};
  position: relative;
  box-shadow: 0 2px 5px 0px rgba(0, 0, 0, 0.43);

  &::before {
    content: '';
    width: 8px;
    height: 8px;
    position: absolute;
    top: -5px;
    left: 43px;
    transform: rotate(45deg);
    z-index: 1;
    background-color: ${colors.white};
    border-top: 1px solid ${colors.blue};
    border-left: 1px solid ${colors.blue};
  }
`;

export const FilterList = styled.ul`
  margin: 0;
  height: 100%;
  list-style: none;
  padding: 0;
  overflow: auto;
`;

export const FilterListItem = styled.li`
  height: 35px;
  display: flex;
  padding: 0 1rem;
  align-items: center;
`;

export const Checkbox = styled.input.attrs(() => ({ type: 'checkbox' }))`
  margin-right: 0.5rem;
`;

export const FilterButton = styled.button`
  background-color: ${colors.blue};
  border: none;
  border-radius: 2px;
  color: ${colors.white};
  font-weight: 300;
  height: 25px;
  min-width: 60px;
  padding: 0 1rem;
`;
