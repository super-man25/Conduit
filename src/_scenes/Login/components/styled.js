import styled from 'styled-components';
import { cssConstants } from '_constants';

export const HideMe = styled.div`
  display: ${(props) => (props.show ? 'block' : 'none')};
  margin: 0;
  padding: 0;
`;

export const ForgotLink = styled.span`
  cursor: pointer;
  color: ${(props) => props.color || cssConstants.PRIMARY_LIGHT_GRAY};
  font-size: ${(props) => props.size || cssConstants.SUBHEADING_SIZE_S1};
  font-weight: ${(props) => props.weight || cssConstants.SUBHEADING_WEIGHT_S1};
  &:hover {
    color: ${(props) => props.color || cssConstants.PRIMARY_LIGHT_BLACK};
  }
`;

export const Content = styled.div`
  height: 100%;
  display: flex;
  flex-direction: column;
  justify-content: space-between;
`;

export const LoginFooter = styled.div`
  padding: 40px;
  background-color: ${cssConstants.PRIMARY_EVEN_LIGHTER_GRAY};
  font-size: 10px;
  line-height: 130%;
`;

export const LogoImg = styled.img`
  display: block;
  margin: auto;
  padding: 40px;
  padding-top: 10px;
  width: 40%;
`;
