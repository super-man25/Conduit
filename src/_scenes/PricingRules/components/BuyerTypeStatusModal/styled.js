import styled from 'styled-components';
import { cssConstants } from '_constants';
import { H4, Flex, Input, Text } from '_components';
import { darken } from 'polished';

export const ModalOverlay = styled.div`
  position: fixed;
  top: 0;
  bottom: 0;
  left: 0;
  right: 0;
  opacity: 0.4;
  background-color: ${cssConstants.PRIMARY_LIGHT_BLACK};
`;

export const ModalContent = styled.div`
  position: fixed;
  top: 50%;
  left: 50%;
  transform: translate(-50%, -50%);
  width: 500px;
  z-index: 9;
  box-shadow: 0 1px 10px rgba(0, 0, 0, 0.27);
`;

export const ModalHeader = styled.div`
  padding: 1.5rem;
  background-color: ${cssConstants.PRIMARY_LIGHTEST_GRAY};
  border-bottom: 1px solid ${cssConstants.PRIMARY_DARK_BLUE};
  border-top-left-radius: 3px;
  border-top-right-radius: 3px;
`;

export const ModalBody = styled.div`
  padding: 1rem 2rem;
  height: 400px;
  overflow-x: hidden;
  overflow-y: scroll;
  background-color: ${cssConstants.PRIMARY_LIGHTEST_GRAY};
  border-bottom: 1px solid ${cssConstants.PRIMARY_DARK_BLUE};
`;

export const ModalFooter = styled.div`
  padding: 1.5rem;
  background-color: ${cssConstants.PRIMARY_LIGHTEST_GRAY};
  border-bottom: ${cssConstants.PRIMARY_DARK_BLUE};
  border-bottom-left-radius: 3px;
  border-bottom-right-radius: 3px;
`;

export const BuyerTypeOption = Flex.extend`
  padding: 0 2rem;
  min-height: 45px;
  width: 100%;
  background-color: ${cssConstants.PRIMARY_LIGHTEST_GRAY};

  border-bottom: 1px solid ${cssConstants.PRIMARY_LIGHTER_GRAY};

  :hover {
    background-color: ${darken(0.05, cssConstants.PRIMARY_WHITE)};
  }
`;

export const FlexText = Text.extend`
  flex-basis: ${(props) => props.basis || '10%'};
`;

export const ErrorText = Text.extend`
  text-align: right;
`;

export const Title = H4.extend`
  margin: 0;
  margin-bottom: 2rem;
  color: ${cssConstants.PRIMARY_LIGHT_BLACK};
`;

export const FilterBuyerTypesInput = Input.extend`
  border-radius: 20px;
  height: 35px;
  width: 80px;
`;
