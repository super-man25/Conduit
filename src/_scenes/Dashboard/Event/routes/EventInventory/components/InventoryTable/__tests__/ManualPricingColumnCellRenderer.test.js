import React from 'react';
import { shallow } from 'enzyme';
import { Text, ClearIcon } from '_components';
import { ManualPricingCellPresenter } from '../ManualPricingColumnCellRenderer';
import { ManualPricingButton } from '../styled';

describe('<ManualPricingColumnCellRenderer />', () => {
  const props = {
    isEditing: false,
    setManualPrice: jest.fn(),
    manualPriceLocked: false,
    setEditingManualPrice: jest.fn(),
    cancelEditingManualPrice: jest.fn(),
    rowData: { listedPrice: 12 },
    cellData: null
  };

  it('should render correctly', () => {
    const wrapper = shallow(<ManualPricingCellPresenter {...props} />);
    expect(wrapper).toMatchSnapshot();
  });

  it('should render correctly when the cellData has a value', () => {
    const wrapper = shallow(
      <ManualPricingCellPresenter {...props} cellData={12} />
    );
    expect(wrapper).toMatchSnapshot();
  });

  it('should render correctly when isEditing is true', () => {
    const wrapper = shallow(
      <ManualPricingCellPresenter {...props} isEditing={true} />
    );
    expect(wrapper).toMatchSnapshot();
  });

  it('should render correctly when manualPriceLocked is true', () => {
    const wrapper = shallow(
      <ManualPricingCellPresenter {...props} manualPriceLocked={true} />
    );
    expect(wrapper).toMatchSnapshot();
  });

  it('should call setEditingManualPrice when the set price text is clicked', () => {
    const mockFn = jest.fn();
    const wrapper = shallow(
      <ManualPricingCellPresenter {...props} setEditingManualPrice={mockFn} />
    );
    wrapper
      .find(Text)
      .first()
      .simulate('click');

    expect(mockFn).toHaveBeenCalled();
  });

  it('should not call setEditingManualPrice when the set price text is clicked if editing is locked', () => {
    const mockFn = jest.fn();
    const wrapper = shallow(
      <ManualPricingCellPresenter
        {...props}
        manualPriceLocked={true}
        setEditingManualPrice={mockFn}
      />
    );
    wrapper
      .find(Text)
      .first()
      .simulate('click');

    expect(mockFn).not.toHaveBeenCalled();
  });

  it('should call cancelEditingManualPrice when the ClearIcon is clicked', () => {
    const mockFn = jest.fn();
    const wrapper = shallow(
      <ManualPricingCellPresenter
        {...props}
        isEditing={true}
        cancelEditingManualPrice={mockFn}
      />
    );
    wrapper
      .find(ClearIcon)
      .first()
      .simulate('click');

    expect(mockFn).toHaveBeenCalled();
  });

  it('should call setManualPrice when the ManualPricingButton is clicked and a value has been set', () => {
    const mockFn = jest.fn();
    const wrapper = shallow(
      <ManualPricingCellPresenter
        {...props}
        isEditing={true}
        setManualPrice={mockFn}
      />
    );

    wrapper.setState({
      overridePrice: '12.00'
    });

    wrapper
      .find(ManualPricingButton)
      .first()
      .simulate('click');

    expect(mockFn).toHaveBeenCalled();
  });
});
