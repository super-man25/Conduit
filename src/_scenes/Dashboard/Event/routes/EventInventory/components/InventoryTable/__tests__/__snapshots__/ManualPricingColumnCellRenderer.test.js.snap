// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ManualPricingColumnCellRenderer /> should render correctly 1`] = `
<Flex
  align="center"
>
  <Styled(styled.p)
    color="#38A9DB"
    cursor="pointer"
    ellipsis={true}
    onClick={[Function]}
    opacity={1}
    size={14}
    title="SET PRICE"
  >
    SET PRICE
  </Styled(styled.p)>
</Flex>
`;

exports[`<ManualPricingColumnCellRenderer /> should render correctly when isEditing is true 1`] = `
<Flex
  align="center"
>
  <NumberInputField
    component={
      Object {
        "$$typeof": Symbol(react.forward_ref),
        "attrs": Array [
          [Function],
        ],
        "componentStyle": ComponentStyle {
          "componentId": "sc-eilVRo",
          "isStatic": false,
          "rules": Array [
            "
  box-sizing: border-box;
  display: block;
  margin-top: 10px;
  margin-bottom: 5px;
  background: ",
            [Function],
            ";
  background-image: ",
            [Function],
            ";
  background-repeat: no-repeat;
  background-position: right center;

  &:-webkit-autofill {
    animation-name: ",
            [Function],
            ";
    animation-fill-mode: both;
  }

  font-size: 1rem;
  width: ",
            [Function],
            ";
  padding-left: 3%;
  border: 2px solid;
  border-radius: 3px;
  border-color: ",
            [Function],
            ";
  padding-top: 0.9em;
  padding-bottom: 0.9em;

  ::placeholder {
    color: ",
            "#999999",
            ";
  }
",
            "
  box-sizing: border-box;
  margin: 0 0.5rem 0 0;
  padding: 10px 8px;
  max-height: 35px;
  max-width: 80px;
",
          ],
        },
        "displayName": "Styled(styled.input)",
        "foldedComponentIds": Array [
          "sc-hEsumM",
        ],
        "render": [Function],
        "styledComponentId": "sc-eilVRo",
        "target": "input",
        "toString": [Function],
        "warnTooManyClasses": [Function],
        "withComponent": [Function],
      }
    }
    onChange={[Function]}
    placeholder="$12.00"
    value=""
  />
  <Styled(Styled(Styled(styled.button)))
    onClick={[Function]}
  >
    Set Price
  </Styled(Styled(Styled(styled.button)))>
  <Flex>
    <ClearIcon
      fill="#000"
      height="24px"
      onClick={[Function]}
      styles={
        Object {
          "cursor": "pointer",
          "marginLeft": 7,
        }
      }
      width="24px"
    />
  </Flex>
</Flex>
`;

exports[`<ManualPricingColumnCellRenderer /> should render correctly when manualPriceLocked is true 1`] = `
<Flex
  align="center"
>
  <Styled(styled.p)
    color="#38A9DB"
    cursor="pointer"
    ellipsis={true}
    onClick={[Function]}
    opacity={0.5}
    size={14}
    title="SET PRICE"
  >
    SET PRICE
  </Styled(styled.p)>
</Flex>
`;

exports[`<ManualPricingColumnCellRenderer /> should render correctly when the cellData has a value 1`] = `
<Flex
  align="center"
>
  <Styled(styled.p)
    color="#38A9DB"
    cursor="pointer"
    marginLeft="1rem"
    onClick={[Function]}
    size={14}
  >
    $12.00
  </Styled(styled.p)>
</Flex>
`;
